app/main.py

```diff
@@ -1,3 +1,57 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int or float) -> None:
```

Comment:
Use `|` insred of `or` in the type annotation here and in the other places as well